libraries/core-react/src/components/Scrim/Scrim.tsx

```diff
@@ -15,45 +15,49 @@ const StyledScrim = styled.div`
   left: 0;
   align-items: center;
   justify-content: center;
-  display: flex;
+  ${({ open }) => css({ display: open ? 'flex' : 'none' })}
 `
 
 const ScrimContent = styled.div`
   width: auto;
   height: auto;
 `
 
+type StyledScrimProps = Pick<ScrimProps, 'open'>
+
 export type ScrimProps = {
   /** Whether scrim can be dismissed with esc key and outside click
    */
   isDismissable?: boolean
+  /** programmatically toggle scrim */
+  open: boolean
   /** function to handle closing scrim */
-  onClose?: (event: MouseEvent | KeyboardEvent, open: boolean) => void
+  onClose?: (event: MouseEvent | KeyboardEvent) => void
```

Comment:
suggest removing `event: MouseEvent | KeyboardEvent` in `onClose` as we don't have any events in our other `onClose` handlers